readme.md

```diff
@@ -1,7 +1,7 @@
 # sbt-scalafmt
 [![Join the chat at https://gitter.im/scalameta/scalafmt](https://badges.gitter.im/scalameta/scalafmt.svg)](https://gitter.im/scalameta/scalafmt?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Join the discord chat](https://img.shields.io/discord/632642981228314653?label=discord)](https://discordapp.com/channels/632642981228314653/632665341864181780)
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.scalameta/sbt-scalafmt/badge.svg?kill_cache=1)](https://search.maven.org/artifact/org.scalameta/sbt-scalafmt/)
+[![Latest version](https://index.scala-lang.org/scalameta/sbt-scalafmt/latest.svg?kill_cache=1)](https://index.scala-lang.org/scalameta/sbt-scalafmt)
```

Comment:
could you also revise the "maven central" reference below? (i found one on line 12)

Reply:
Done!
I pushed a new commit because I didn't want to force-push. Let me know your preferences if you have any.